src/p_local.h

```diff
@@ -387,6 +387,10 @@ void	P_PlaySpawnSound(AActor *missile, AActor *spawner);
 // [RH] Position the chasecam
 void	P_AimCamera (AActor *t1, DVector3 &, DAngle &, sector_t *&sec, bool &unlinked);
 
+double	P_GetRadiusDamage(bool fromaction, AActor *bombspot, AActor *thing, int bombdamage, int bombdistance, int fulldamagedistance, bool thingbombsource = false);
+
+int		P_GetOldRadiusDamage(bool fromaction, AActor *bombspot, AActor *thing, int bombdamage, int bombdistance, int fulldamagedistance);
```

Comment:
These functions should not be exposed in header file because they are used in the same souce file where they are defined.
Make them static and move definition of action function below to avoid forward declarations.

Reply:
Done.